admin/client.py

```diff
@@ -1,176 +1,164 @@
 # Copyright Hybrid Logic Ltd.  See LICENSE file for details.
 """
-Run the acceptance tests.
+Run the client installation tests.
 """
 
+import os
+import shutil
 import sys
+import tempfile
 import yaml
 
-from zope.interface import Interface, implementer
-from characteristic import attributes
-from eliot import add_destination
-from twisted.internet.error import ProcessTerminated
+import docker
+from effect import TypeDispatcher, sync_performer, perform
 from twisted.python.usage import Options, UsageError
 from twisted.python.filepath import FilePath
-from twisted.internet.defer import inlineCallbacks, returnValue
 
-from flocker.common.version import make_rpm_version
-from flocker.provision import PackageSource, CLOUD_PROVIDERS
 import flocker
-from flocker.provision._ssh import (
-    run_remotely)
+from flocker.common.version import make_rpm_version
+from flocker.provision import PackageSource
+from flocker.provision._effect import Sequence, perform_sequence
 from flocker.provision._install import (
+    task_install_cli,
     task_client_installation_test,
-    install_cli,
 )
-from effect.twisted import perform
-from flocker.provision._ssh._conch import make_dispatcher
+from flocker.provision._ssh._conch import perform_sudo, perform_put
```

Comment:
I've gone halfway on this, moving the non-conch dependent performers in the `_conch` module into the `_model` module, and then making that importable through `__init.py__` (`from flocker.provision._ssh import...`)